swagger.yml

```diff
@@ -1200,6 +1200,14 @@ paths:
         type: string
         in: path
         required: true
+      - name: buildbot_version
+        type: string
+        in: query
```

Comment:
Shouldn't this be in the body? (Same with the one below)

Reply:
It's in the query intentionally, so we can spot it in our logs (you can never have enough info!). What'd be the benefit from putting it into the body?

Reply:
I think it's a bit nicer and cleaner being a `POST` request, specially if we think of passing more data later. But not a strong opinion, we can always change it if we need to.

Reply:
Since these aren't required, and they're more of a meta information piece than the subject of the `POST`, I prefer keeping them in `query` 🤷 Let's go with this for now :)